pkg/devfile/parser/parse_test.go

```diff
@@ -4317,6 +4318,52 @@ func Test_parseFromRegistry(t *testing.T) {
 	}
 }
 
+func Test_getDevfileFromDir(t *testing.T) {
+	tempDir, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Errorf("Failed to create temp dir: %s, error: %v", tempDir, err)
+	}
+	defer os.RemoveAll(tempDir)
+
+	devfile := filepath.Join(tempDir, "devfile.yaml")
+	if err := ioutil.WriteFile(devfile, []byte(""), 0666); err != nil {
```

Comment:
can you use the fakefs in `testingutil` instead? should avoid creating real files or dir in unit tests